docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -36,7 +36,7 @@ The managed node also needs a user account that can connect through SSH to the n
 Node requirement summary
 ========================
 
-You can find details about control and managed node requirements for each Ansible version in the :ref:`support_life`.
+You can find details about control and managed node requirements, including Python versions, for each Ansible version in the :ref:`support_life` and :ref:`target_node_python_support` sections.
```

Comment:
Consider ansible_core_support_matrix` instead as it points directly to the table maintained by core folks.  This link ignores the control node and might be confusing.